napari/layers/image/image.py

```diff
@@ -534,6 +534,7 @@ def _update_slice_response(self, response: _ImageSliceResponse) -> None:
         """Update the slice output state currently on the layer. Currently used
         for both sync and async slicing.
         """
+        response = response.to_displayed(self._raw_to_displayed)
```

Comment:
Maybe worth a comment that this is needed in analogous fashion to labels?
And update docstring of `to_displayed` in`_slice/_ImageSliceResponse`